README.md

```diff
@@ -2,31 +2,378 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Discuss the `String` class
+- Introduce the String Pool and new `String` objects
+- Discuss some of the `String` class' methods
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+The `String` data type is one of the first data types we started using! But up
+until now, we haven't really talked about the `String` class itself. We already
+know that it is an immutable reference type and can be made up of a sequence of
```

Comment:
Maybe remove the "immutable" word here and define immutable a little more with an example below?

Reply:
The introductory character&string  lesson was updated to define immutable.